### PR TITLE
MAUI-3959 : Added API reference links for tile layer

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
Hi @vigneshrameshsync 
I have added a API reference links for new APIs in tile layer. Could you please review it?

Regards,
Shalini Ashokan.

### CODE REVIEW

File: MAUI/Maps/migration.md [hunk 1]
```diff
@@ -363,47 +363,47 @@ The following code example explains how to intialize the layer and customize the
       {{'[LayerType](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_LayerType)'| markdownify }}, {{'[BingMapKey](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_BingMapKey)'| markdownify }}, {{'[BingMapStyle](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_BingMapStyle)'| markdownify }}, and {{'[RequestTileUri](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_RequestTileUri)'| markdownify }}
    </td>
    <td>
-      Can achieve all this using {{'[UrlTemplate]()'| markdownify }} and {{'[GetBingUrl()]()'| markdownify }}
+      Can achieve all this using {{'[UrlTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_UrlTemplate)'| markdownify }} and {{'[GetBingUrl()]()'| markdownify }}
```
Comment (vigneshrameshsync) [inline MAUI/Maps/migration.md:366]: GetBingUrl() API link missing

File: MAUI/Maps/migration.md [hunk 2]
```diff
@@ -363,47 +363,47 @@ The following code example explains how to intialize the layer and customize the
...
+      {{'[DistanceType](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_DistanceType)'| markdownify }}
    </td>
 </tr>
 <tr>
    <td>
       {{'[LatLngBounds](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_LatLngBounds)'| markdownify }} 
    </td>
    <td>
-      {{'[*MapLatLngBounds]()'| markdownify }} 
+      {{'[*MapLatLngBounds](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_MapLatLngBounds)'| markdownify }}
```
Comment (vigneshrameshsync) [inline MAUI/Maps/migration.md:406]: As of now, just change the property to `LatLngBounds`. We can change the API link later.

File: MAUI/Maps/zoom-pan.md [hunk 3]
```diff
@@ -83,9 +83,9 @@ this.Content = map;
 
 ## Customizing the zoom level
 
-You can set the zoom level of the map layer by using the `MapZoomPanBehavior.ZoomLevel` property.
+You can set the zoom level of the map layer by using the [`MapZoomPanBehavior.ZoomLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapZoomPanBehavior.html#Syncfusion_Maui_Maps_MapZoomPanBehavior_ZoomLevel) property.
 
-The default `MapZoomPanBehavior.ZoomLevel` value is 1, which will show the whole map in the viewport for `MapTileLayer`.
+The default [`MapZoomPanBehavior.ZoomLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapZoomPanBehavior.html#Syncfusion_Maui_Maps_MapZoomPanBehavior_ZoomLevel) value is 1, which will show the whole map in the viewport for `MapTileLayer`.
```
Comment (vigneshrameshsync) [inline MAUI/Maps/zoom-pan.md:88]: `MapTileLayer` link missing

File: MAUI/Maps/migration.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Added API link

File: MAUI/Maps/zoom-pan.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/zoom-pan.md]: Added API link

File: MAUI/Maps/migration.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Changed as `LatLngBounds`

**Review by vigneshrameshsync — APPROVED** [review]
Fine.

**Pull request merged**